src/sdk/pynni/nni/smac_tuner/smac_tuner.py

```diff
@@ -170,13 +172,19 @@ def generate_multiple_parameters(self, parameter_id_list):
         '''
```

Comment:
should also update the function `generate_parameters`.

Reply:
I will add it. But it seems that nni will never invoke generate_parameters actually if you have overwrite generate_parameters already

Reply:
yes, but still we should make `generate_parameters` correct.